src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -429,6 +430,34 @@ private IDataView GetShuffledData(string path)
                         }
                     }
 
+                    //Process last incomplete batch
+                    if (batchIndex > 0)
+                    {
+                        runner.AddInput(new Tensor(featureBatchPtr, featureTensorShape, TF_DataType.TF_FLOAT, featureBatchSizeInBytes), 0)
+                                .AddInput(new Tensor(labelBatchPtr, labelTensorShape, TF_DataType.TF_INT64, labelBatchSizeInBytes), 1)
+                                .Run();
+
```

Comment:
At this point we know the batch size is not equal to batchSize so we will need to change the shape(featureTensorShape,labelTensorShape) of the input tensor so TF does not process the entire batch and same for batchSizeInBytes.

Reply:
done.